web/libs/editor/src/components/Comments/Comment/CommentForm.tsx

```diff
@@ -114,36 +127,69 @@ export const CommentForm: FC<CommentFormProps> = observer(({ commentStore, annot
 
   const currentLinkingComment = annotationStore.selected.currentLinkingMode?.comment;
   const currentComment = getCurrentComment();
-  const { text = "", regionRef } = currentComment || {};
+  const { text = "", regionRef, classifications } = currentComment || {};
   const { region, result } = regionRef || {};
   const linking = !!linkingComment && currentLinkingComment === linkingComment && globalLinking;
   const hasLinkState = linking || region;
+  const selections = taxonomyPathsToSelectedItems(classifications?.default?.values);
+  const classificationsItems = parseCommentClassificationConfig(commentStore.commentClassificationConfig);
+
+  const Buttons = () => (
+    <Elem name="buttons">
+      {!region && (
+        <Tooltip title="Link to..." mouseEnterDelay={TOOLTIP_DELAY}>
+          <Elem name="action" tag="button" mod={{ highlight: linking }} onClick={linkToHandler}>
+            <IconCommentLinkTo />
+          </Elem>
+        </Tooltip>
+      )}
+      <Elem name="action" tag="button" type="submit">
+        <IconSend />
+      </Elem>
+    </Elem>
+  );
```

Comment:
Component defined in a component is usually a sign to lift it out, which ensures that all components remain easy to read through.